### PR TITLE
Thrown pies blind targets

### PR DESCRIPTION
**Pull request by RealestEstate**
If a thrown pie hits a mob, it will cover their face in pie goop and blind them for a couple of seconds. Pies make a cartoon sounding splat. it's a really funny sound bite.
Untitled

:cl:
 * rscadd: Pies blind mobs for a couple of seconds when thrown.

### CODE REVIEW

**Comment by cloutcode** [comment]
had enough of this goddamned clowncreep

**Comment by PrimeDSS13** [comment]
I can't get enough of this goddamned clowncreep

**Comment by Kharkov72** [comment]
more like chefcreep
botany will be reminded of their place

**Comment by Dacendeth** [comment]
Does it work on borgs and if it doesnt why not?

**Comment by tuypo1** [comment]
> had enough of this goddamned clowncreep

clowncreep is a good thing because it leaves people less likely to tolerate clown shittery
but then that attitude is not all that different to what anti cult people do in downvoting cult fixes so it's probably not a great attitude for me to have

**Comment by RealestEstate** [comment]
yeah I tested it locally and it worked on borgs, and in case that it ever doesn't, the intent is for it to work on borgs.

**Comment by tayachiya** [comment]
Does it affect the wiz spell as well?

**Comment by RealestEstate** [comment]
yes

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -1150,10 +1150,26 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/pie/throw_impact(atom/hit_atom)
 	..()
+	if(ismob(hit_atom))
+		var/mob/M = hit_atom
+		src.visible_message("<span class='warning'>\The [src] splats in [M]'s face!</span>")
+
+		M.eye_blind = 2
+		M.overlays += image('icons/mob/messiness.dmi',icon_state = "pied")
+		spawn(55)
```
Comment (RealestEstate) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: should I be using sleep instead? I forget which does which

Comment (RealestEstate) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: ah yes I do

**Pull request merged**